specs/deneb/beacon-chain.md

```diff
@@ -187,6 +188,19 @@ def kzg_commitment_to_versioned_hash(kzg_commitment: KZGCommitment) -> Versioned
     return VERSIONED_HASH_VERSION_KZG + hash(kzg_commitment)[1:]
 ```
 
+#### `is_valid_merkle_path`
+
+```python
+def is_valid_merkle_path(leaf: Bytes32, branch: Sequence[Bytes32], gindex: int, root: Root) -> bool:
+    value = leaf
+    for i in range(len(branch)):
+        if (gindex >> i) & 1 == 0:
```

Comment:
Is this a bug? I feel that this should be `if (gindex >> i) & 1 == 1` instead. Even if `gindex` is a generalized merkle tree index, I think, if it's even, it will be on the left of its parent, and, if it's odd, it will be on the right.